### DISCUSSION TITLE
Skip TikTok Shop videos

### DISCUSSION THREAD

**Discussion by Stavrak2** [dayanch96/BHTikTok-Plus#16] [Ideas]
Would be great if we could skip over any video that contains a TikTok shop link. And hide “Your Orders” on our own profile page.

**Comment by dayanch96** [comment]
Hi. I don't have US version of TikTok and can't test it without having ;(

**Reply by Stavrak2** [reply]
Can’t you do it with “change region” and register a US account?
Possibly in combination with disable mobile data and put on US VPN?

**Reply by dayanch96** [reply]
nope it doesn’t work like that

**Reply by Stavrak2** [reply]
If you want to get the version specific to the US App Store (I’ve tried this and it let me install multiple copies from different regions):

go to Appleid.apple.com
Register a new Apple ID under that country, use fake details et cetera, it doesn’t need a phone or credit card
Going to the iCloud then media and purchases
You can temporarily log out there without having to log out of iCloud completely, sign in using the new account you made, you will, then have that countries App Store, where you can download as many of the countries apps as you want, then just come back to the media and purchases section and log back in Using your normal account

You will now have 2 TikTok’s - your home country’s one and the new foreign version. they will exist and update separately. 

You might need to make a US TikTok account to go with it, of course
